README.md

```diff
@@ -117,23 +116,23 @@ results = (
 ```
 <img src="towhee_example.png" style="width: 60%; height: 60%">
 
-Learn more examples from [Towhee bootcamp](https://codelabs.towhee.io/)
+Learn more examples from the [Towhee Bootcamp](https://codelabs.towhee.io/).
 
 ## Core Concepts
 
 Towhee is composed of four main building blocks - `Operators`, `Pipelines`, `DataCollection API` and `Engine`.
 
-- __Operator__: An operator is a single building block of neural data processing pipelines. Different implementations of operators are categorized by tasks, with standard task interface. An operator can be a deep learning model, a data processing method, or a Python function.
+- __Operators__: An operator is a single building block of neural data processing pipelines. Different implementations of operators are categorized by tasks, with a standard task interface. An operator can be a deep learning model, a data processing method, or a Python function.
 
-- __Pipeline__: A pipeline is composed of several operators. Operators are connected together as a DAG(directed acyclic graph) to create complex functionalities, such as embedding feature extraction, data tagging, cross modal data understanding, etc.
+- __Pipelines__: A pipeline is composed of several operators. Operators are connected together as a DAG (directed acyclic graph) to create complex functionalities, such as embedding feature extraction, data tagging, and cross modal data understanding.
 
-- __DataCollection__: A pythonic and method-chaining style API that for building custom pipelines. Pipelines defined by DataColltion API can be either run on notebook locally for fast prototyping, or converting to image docker with end-to-end optimization for production-ready environments. 
+- __DataCollection API__: A Pythonic and method-chaining style API for building custom pipelines. Pipelines defined by DataColltion API can run on laptops locally for fast prototyping, or can be converted to docker images with end-to-end optimization for production-ready environments.
```

Comment:
- __DataCollection API__: A Pythonic and method-chaining style API for building custom pipelines. A pipeline defined by the DataColltion API can be run locally on a laptop for fast prototyping and then be converted to a docker image, with end-to-end optimizations, for production-ready environments.